shared/message/support/Gazebo/GazeboWorldCtrl.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
+
+package message.support.Gazebo;
+
+message GazeboWorldCtrl {
```

Comment:
Sending opaque strings is a terrible way to communicate. Also abbreviating Control to Ctrl is not that much of a saving